history_test.go

```
@@ -0,0 +1,127 @@
+package merkletree
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/coniks-sys/libmerkleprefixtree-go/crypto"
+)
+
+var signKey, _ = crypto.GenerateKey()
```

Let's put this in an `init()` and not ignore the error.